v0/webhook/transform.js

```diff
@@ -0,0 +1,35 @@
+const {
+  defaultPostRequestConfig,
+  defaultRequestConfig,
+  isURL
+} = require("../util");
+
+function process(event) {
+  const { message, destination } = event;
+  const response = defaultRequestConfig();
+  if (destination.Config.webhookUrl && isURL(destination.Config.webhookUrl)) {
+    response.endpoint = destination.Config.webhookUrl;
+    response.method = defaultPostRequestConfig.requestMethod;
+    // add content-type by default as our payload is json
+    response.headers = {
+      "Content-Type": "application/json"
+    };
+
+    const configHeaders = destination.Config.header;
```

Comment:
Should we change the config-backend param to headers as well (instead of header)

Reply:
Updated both control-plane and here